npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -2,31 +2,53 @@
   margin-bottom: 40px;
 }
 
+.drag-and-drop__button {
+  margin-bottom: 32px;
+  margin-top: 65px;
+  padding-left: 15px;
+  position: static;
+  width: auto;
+}
+
+.drag-and-drop__button::before {
+  position: static;
+}
+
+.drag-and-drop__button--highlight {
+  background-color: $govuk-focus-colour;
+  box-shadow: 0 -2px $govuk-focus-colour, 0 4px govuk-colour('black');
+  color: $govuk-text-colour;
+  outline: none;
+}
+
+.drag-and-drop--dragover {
+  background: #b1b4b6;
```

Comment:
This colour is in the [palette](https://design-system.service.gov.uk/styles/colour/). The outline colour isn't - is there a similar colour that we could use instead?

Reply:
Cool, I'll use this `govuk-colour("dark-grey")`; it's probably the closest.